packages/core/content-manager/admin/src/hooks/tests/useDocument.test.tsx

```diff
@@ -329,6 +332,40 @@ describe('useDocument', () => {
      */
     expect(result.current.components['profiles.image']).toBeUndefined();
   });
+
+  it('should return a getTitle function', async () => {
+    const { result } = renderHook(() =>
+      useDocument({
+        collectionType: 'collection-types',
+        model: mockData.contentManager.contentType,
+        documentId: '12345',
+      })
+    );
+
+    await waitFor(() => expect(result.current.isLoading).toBe(false));
+
+    expect(result.current.getTitle).toBeInstanceOf(Function);
```

Comment:
is this assertion needed to check the next assertion?

Reply:
No it's not needed I just saw some other tests also checked this, so stayed consistent 🤷‍♂️